### PR TITLE
Add doc for native config of testcafe

### PR DESCRIPTION
**Pull request by tianfeng92**
**Thank you for contributing to the `sauce-docs` project!**
**A well described pull request helps maintainers quickly review and merge your change**

Before submitting your PR, please check our [contributing](https://github.com/saucelabs/sauce-docs/blob/master/CONTRIBUTING.MD)
guidelines, applied for this repository. Avoid large PRs, help reviewers by making them as simple 
and short as possible.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [x] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/master/CONTRIBUTING.MD) document.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/testrunner-toolkit/configuration/testcafe.md [hunk 1]
```diff
@@ -65,13 +106,101 @@ __Example__:
...
+```yaml
+  src:
+    - "tests/test_file1.test.js"
+    - "tests/integrations"
+    - "*/*.test.js"
+```
+
+### `env`
+
+__Description__: Environment variables. It can be expanded.
```
Comment (dpgraham) [inline docs/testrunner-toolkit/configuration/testcafe.md:136]: Could we elaborate on what it means to "expand" environment variables.

File: docs/testrunner-toolkit/configuration/testcafe.md [hunk 2]
```diff
@@ -65,13 +106,101 @@ __Example__:
...
+__Example__:
+```yaml
+  env:
+    hello: world
+    foo: $my_bar
+```
+
+### `screenshots`
+
+__Description__: Native screenshots settings for testcafe.
```
Comment (dpgraham) [inline docs/testrunner-toolkit/configuration/testcafe.md:149]: How about `Screenshots settings for testcafe. See <link to testcafe>`

File: docs/testrunner-toolkit/configuration/testcafe.md [hunk 3]
```diff
@@ -65,13 +106,101 @@ __Example__:
...
+```yaml
+    testMatch: '**/*.js'
+```
+
+### `screenResolution`
+
+<p><small><Highlight color="#ad1415">sauce cloud only</Highlight></small><a href="/testrunner-toolkit/running-tests#test-on-sauce-labs">ℹ</a></p>
 
-__Description__: Regular expression for locating test files.
+__Description__: Field where you can change the browser window screen resolution.
```
Comment (dpgraham) [inline docs/testrunner-toolkit/configuration/testcafe.md:199]: Maybe we can shorten this to just `Set browser window screen resolution`

File: docs/testrunner-toolkit/configuration/testcafe.md (hunk 1 above)
Comment (tianfeng92) [reply docs/testrunner-toolkit/configuration/testcafe.md:136]: sure.

Comment (tianfeng92) [reply docs/testrunner-toolkit/configuration/testcafe.md:136]: sure.

File: docs/testrunner-toolkit/configuration/testcafe.md (hunk 2 above)
Comment (tianfeng92) [reply docs/testrunner-toolkit/configuration/testcafe.md:149]: good idea!

File: docs/testrunner-toolkit/configuration/testcafe.md (hunk 3 above)
Comment (tianfeng92) [reply docs/testrunner-toolkit/configuration/testcafe.md]: updated

**Review by spider-sauce — APPROVED** [review]

File: docs/testrunner-toolkit/configuration/testcafe.md [hunk 4]
```diff
@@ -65,13 +106,101 @@ __Example__:
   - name: "saucy test"
 ```
 
-### `match`
+### `browserName`
+
+__Description__: Name of desired browser. Although Testcafe supports triggering one test in multiple browsers, it'd better here to split them into every suite to indicate each suite has its own test point.
```
Comment (spider-sauce) [inline docs/testrunner-toolkit/configuration/testcafe.md:111]:
```suggestion
__Description__: Name of desired browser. Although Testcafe supports triggering one test in multiple browsers, it is better here to split them into every suite to indicate each suite has its own test point.
```

File: docs/testrunner-toolkit/configuration/testcafe.md [hunk 5]
```diff
@@ -10,48 +10,89 @@ Please refer to the [Common Configuration Syntax Reference](/testrunner-toolkit/
...
+
+__Example__:
+```yaml
+  projectPath: /path/to/tests/
 ```
 
 ## `suites`
 
-__Description__:
+__Description__: Field for defining test suite details such as the suite `name`, desired `browserName` and test configurations.
```
Comment (spider-sauce) [inline docs/testrunner-toolkit/configuration/testcafe.md:81]:
```suggestion
__Description__: Field for defining test suite details such as the suite `name`, desired `browserName`, and test configurations.
```

File: docs/testrunner-toolkit/configuration/testcafe.md [hunk 6]
```diff
@@ -10,48 +10,89 @@ Please refer to the [Common Configuration Syntax Reference](/testrunner-toolkit/
...
 __Example__:
 ```yaml
-files:
-  - ./tests/testcafe/
+  version: ##VERSION##
+```
+
+### `projectPath`
+
+__Description__: Absolute path to the test directory and related test files
```
Comment (spider-sauce) [inline docs/testrunner-toolkit/configuration/testcafe.md:70]:
```suggestion
__Description__: Absolute path to the test directory and related test files.
```

File: docs/testrunner-toolkit/configuration/testcafe.md [hunk 7]
```diff
@@ -10,48 +10,89 @@ Please refer to the [Common Configuration Syntax Reference](/testrunner-toolkit/
...
+docker:
+  fileTransfer: mount
+testcafe:
+  projectPath: tests/e2e/testcafe
+  version: 1.8.5
+```
+
+## `testcafe`
+
+__Description__: Details specific to the `testcafe` project configuration
```
Comment (spider-sauce) [inline docs/testrunner-toolkit/configuration/testcafe.md:46]:
```suggestion
__Description__: Details specific to the `testcafe` project configuration.
```

File: docs/testrunner-toolkit/configuration/testcafe.md [hunk 8]
```diff
@@ -65,13 +106,101 @@ __Example__:
...
+__Type__: *string*
+
+__Example__:
+```yaml
+  browserName: "chrome"
+```
+
+### `src`
+
+__Description__: The explicit name, file glob, or location of the test files
```
Comment (spider-sauce) [inline docs/testrunner-toolkit/configuration/testcafe.md:122]:
```suggestion
__Description__: The explicit name, file glob, or location of the test files.
```

File: docs/testrunner-toolkit/configuration/testcafe.md [hunk 9]
```diff
@@ -65,13 +106,101 @@ __Example__:
...
+```yaml
+  src:
+    - "tests/test_file1.test.js"
+    - "tests/integrations"
+    - "*/*.test.js"
+```
+
+### `env`
+
+__Description__: Environment variables. Substituted variables like $MY_VAR can be expanded
```
Comment (spider-sauce) [inline docs/testrunner-toolkit/configuration/testcafe.md:136]:
```suggestion
__Description__: Environment variables. Substituted variables like $MY_VAR can be expanded.
```

File: docs/testrunner-toolkit/configuration/testcafe.md [hunk 10]
```diff
@@ -65,13 +106,101 @@ __Example__:
...
+__Example__:
+```yaml
+  env:
+    hello: world
+    foo: $MY_BAR
+```
+
+### `screenshots`
+
+__Description__: Screenshots settings for testcafe. [See link in Testcafe](https://devexpress.github.io/testcafe/documentation/reference/configuration-file.html#screenshots)
```
Comment (spider-sauce) [inline docs/testrunner-toolkit/configuration/testcafe.md:149]:
```suggestion
__Description__: Screenshots settings for testcafe. [See link in Testcafe](https://devexpress.github.io/testcafe/documentation/reference/configuration-file.html#screenshots).
```

File: docs/testrunner-toolkit/configuration/testcafe.md [hunk 11]
```diff
@@ -65,13 +106,101 @@ __Example__:
...
+__Type__: *string*
+
+__Example__:
+```yaml
+    platformName: "Windows 10"
+```
+
+### `testMatch`
+
+__Description__: The explicit name, regex, or location of the test files
```
Comment (spider-sauce) [inline docs/testrunner-toolkit/configuration/testcafe.md:186]:
```suggestion
__Description__: The explicit name, regex, or location of the test files.
```

File: docs/testrunner-toolkit/configuration/testcafe.md [hunk 12]
```diff
@@ -65,13 +106,101 @@ __Example__:
...
+```yaml
+    testMatch: '**/*.js'
+```
+
+### `screenResolution`
+
+<p><small><Highlight color="#ad1415">sauce cloud only</Highlight></small><a href="/testrunner-toolkit/running-tests#test-on-sauce-labs">ℹ</a></p>
 
-__Description__: Regular expression for locating test files.
+__Description__: Set browser window screen resolution
```
Comment (spider-sauce) [inline docs/testrunner-toolkit/configuration/testcafe.md:199]:
```suggestion
__Description__: Set browser window screen resolution.
```

**Comment by spider-sauce** [comment]
@tianfeng92 LGTM! :shipit:

**Comment by tianfeng92** [comment]
@spider-sauce cool! thanks!
